extra_geom/detectors.py

```diff
@@ -1020,7 +1020,10 @@ def from_origin(cls, origin=(0, 0), asic_gap=2, panel_gap=(16, 30),
 
         # coordinates relative to the first pixel of the first module
         # detector's bottom-right corner
-        ref = (0, - (3 * (cls.frag_fs_pixels + panel_gap[1]) * unit))  
+        ref = (
+            asic_gap_px * cls.pixel_size - module_width,      # x
```

Comment:
You define module width as 8 tiles plus 8 asic (tile) gaps, so here you're subtracting the extra gap. I'd say module width should be 8 tiles plus *7* asic gaps, so this would just be `- module_width`. Then panel_gap_x would be from the end of one module to the start of its neighbour, without projecting an extra asic gap into it.

Reply:
Thanks, yes that makes sens and I changed it